src/Ocelot/WebSockets/WebSocketsProxyMiddleware.cs

```diff
@@ -1,26 +1,33 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 // Modified https://github.com/aspnet/Proxy websockets class to use in Ocelot.
 
 using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration;
 using Ocelot.Logging;
 using Ocelot.Middleware;
 using System.Net.WebSockets;
 
-namespace Ocelot.WebSockets.Middleware
+namespace Ocelot.WebSockets
 {
     public class WebSocketsProxyMiddleware : OcelotMiddleware
```

Comment:
- This class should be removed at all in favor of [WebSocketMiddleware](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.websockets.websocketmiddleware?view=aspnetcore-7.0) by Microsoft .NET team.
- This class should be redesigned based on [WebSocketMiddleware](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.websockets.websocketmiddleware?view=aspnetcore-7.0) by Microsoft .NET team.